sql/core/benchmarks/CollationBenchmark-results.txt

```diff
@@ -1,54 +1,54 @@
+UTF8_BINARY                                       12370          12383          19          0.0      123697.8       1.0X
+UTF8_LCASE                                        28164          28166           3          0.0      281639.6       2.3X
+UNICODE                                          375367         375482         162          0.0     3753668.4      30.3X
+UNICODE_CI                                       418811         419177         517          0.0     4188111.1      33.9X
 
-OpenJDK 64-Bit Server VM 17.0.12+7-LTS on Linux 6.5.0-1025-azure
+OpenJDK 64-Bit Server VM 17.0.13+11-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - startsWith:   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        6844           6848           5          0.0       68440.4       1.0X
-UTF8_LCASE                                        21849          21870          30          0.0      218486.3       3.2X
-UNICODE                                          363474         363811         476          0.0     3634738.4      53.1X
-UNICODE_CI                                       427563         428029         659          0.0     4275629.8      62.5X
+UTF8_BINARY                                       11261          11271          15          0.0      112607.7       1.0X
+UTF8_LCASE                                        26357          26366          12          0.0      263570.5       2.3X
+UNICODE                                          363306         364001         983          0.0     3633059.7      32.3X
+UNICODE_CI                                       420866         421292         603          0.0     4208656.7      37.4X
 
-OpenJDK 64-Bit Server VM 17.0.12+7-LTS on Linux 6.5.0-1025-azure
+OpenJDK 64-Bit Server VM 17.0.13+11-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - endsWith:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        6904           6907           4          0.0       69039.3       1.0X
-UTF8_LCASE                                        22007          22009           3          0.0      220067.8       3.2X
-UNICODE                                          376402         377858        2060          0.0     3764015.4      54.5X
-UNICODE_CI                                       444485         444809         458          0.0     4444850.8      64.4X
+UTF8_BINARY                                       12000          12018          26          0.0      120000.9       1.0X
```

Comment:
Relatively, `UTF8_BINARY` becomes slower than before. Do you happen to know any reason?

Reply:
Let me run the benchmark a few more times.